src/css/variables.scss

```diff
@@ -6,4 +6,11 @@ $smoky-black: #0A0908;
 $pearl: #EAE0D5;
 
 $title-fonts: 'Merriweather';
-$not-title-fonts: 'Montserrat';
\ No newline at end of file
+$not-title-fonts: 'Montserrat';
+
+//Font things
+h2, h4 {
```

Comment:
This should not be here. If you want to have fonts in a different file then have a fonts.scss or move this to styles.scss